trigger/task.go

```diff
@@ -151,6 +151,12 @@ func (t *taskTriggers) Fetch(e *event.EventLogEntry) error {
 	if t.task == nil {
 		return errors.New("couldn't find task")
 	}
+	if t.task.DisplayOnly {
```

Comment:
Should this logic be put in MergeNewTestResults instead?

Reply:
Doh yea, brain fart